api-lib/HttpError.ts

```diff
@@ -76,22 +76,25 @@ export class UnprocessableError extends BaseHttpError {
   details?: any;
 }
 
-export function errorResponse(res: VercelResponse, error: any): void {
```

Comment:
this sneaky bug was making my test fail, because the handler was returning from `await` before finishing sending the error response